roles/setup_venv_app/tasks/main.yml

```diff
@@ -8,7 +8,10 @@
   command: "{{ virtualenv_cmd }} '{{ base_dir }}'"
 
 - name: Clone Git repository
-  command: "git clone {{ git_repo }} '{{ base_dir }}/appdata'"
+  git:
+    repo: https://emergingplatforms.visualstudio.com/RoboAdvisor/_git/RoboAdvisorBot
```

Comment:
I think you leaked your personal repo name here. That should still be `'{{ git_repo }}'`